Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -237,6 +237,15 @@ input[type=submit], input[type=button], input[type=reset], input[type=checkbox],
     cursor: unset;
 }
 
+input[type=button] {
```

Comment:
Why doesn't this handle submit and reset?

Reply:
This was an oversight and is fixed now.